server/api/test/index.js

```diff
@@ -117,4 +117,24 @@ describe( 'api', function() {
 			.send( { username: 'validuser', password: 'validpassword' } )
 			.expect( 408, response, done );
 	} );
+
+	maybeIt( 'should return a 400 push verification request if the server supports it', function( done ) {
```

Comment:
Yes, I noticed this mixing of terms in the config options, too -- some of them are [prefixed with `desktop_oauth_`](https://github.com/gyoshev/wp-calypso/blob/067035345cb3aa431f0fa089b4a6c1e11e8b15c2/server/api/oauth.js#L14-L16) where `oauth_`would be sufficient. If WP.com Desktop has been the first OAuth Calypso client, it may explain the specificity.

Since this appears to be only used in `server/api/oauth.js`, and changing it does not break the tests, I believe I should remove the `desktop_` prefix. It will also remove the duplication of config options (there is now `oauth_client_id` and `desktop_oauth_client_id`). The original commit is prior to the open-sourcing of Calypso, so any insight if this change will break something is welcome.